SmartEVSE2.X/EVSE.c

```diff
@@ -516,7 +517,6 @@ void read_settings(void) {
     eeprom_read_object(&EMConfig[EM_CUSTOM].ERegister, sizeof EMConfig[EM_CUSTOM].ERegister);
     eeprom_read_object(&EMConfig[EM_CUSTOM].EDivisor, sizeof EMConfig[EM_CUSTOM].EDivisor);
     eeprom_read_object(&EMConfig[EM_CUSTOM].DataType, sizeof EMConfig[EM_CUSTOM].DataType);
-    eeprom_read_object(&EMConfig[EM_CUSTOM].Function, sizeof EMConfig[EM_CUSTOM].Function);
```

Comment:
For backward compatibility the order in eeprom_read/write must not be changed - otherwise after a firmware update the values will be mixed up...